components/css/newsletter-text-only/newsletter-text-only-1.0.0.css

```diff
@@ -3407,8 +3410,18 @@ form.purdueForm .robotic {
     line-height: 1em;
     text-align: center;
     padding-top: 0;
-    padding-bottom: 1.5em; }
-  .cta .btn-center {
-    margin: auto !important;
-    display: block;
-    width: 250px; }
+    padding-bottom: 30px; }
+
+.btn-center {
+  display: block;
+  margin: auto !important;
+  margin-bottom: 30px;
+  width: 250px; }
+
+@media (min-width: 768px) {
+  .intro__description {
+    margin: 0 auto !important;
+    width: 75%; }
+
+  .bgfix {
+    background-color: #2e2e2e; } }
```

Comment:
This looks like another build that is located in the components folder. Lets go ahead and delete this file. Our workflow should automatically place this build in the "builds" folder.